src/message.rs

```diff
@@ -1059,7 +1061,7 @@ impl Lot {
                 );
                 self.text1_meaning = Meaning::Text1Self;
             }
-        } else if chat.typ == Chattype::Group {
+        } else if chat.typ == Chattype::Group || chat.typ == Chattype::Mailinglist {
```

Comment:
This could be a `match` with an explicit branch for `Single` filling `None` into `self.text1`.